src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1365,6 +1366,17 @@ class ContentExplorer extends Component<Props, State> {
         }
     };
 
+    /**
+     * Returns whether the currently focused element is an item
+     *
+     * @returns {bool}
+     */
+    isFocusedOnItem = () => {
+        const focusedClassList = [...(document.activeElement?.classList || [])];
+        const itemClassList = ['be-item-label'];
+        return itemClassList.every(itemClass => focusedClassList.includes(itemClass));
```

Comment:
if we're only checking one class now, we could do:
```suggestion
        return focusedClassList.includes('be-item-label');
```

Reply:
ah wait ur right LOL